R/meta_regression/indirect_contribution_plot.R

```diff
@@ -176,7 +163,7 @@ CreateIndirectContributionPlot <- function(
   
   contribution_df <- data.frame(
     Treatment = treatments,
-    covariate_value = covariate_values,
+    covariate_value = covariate_values + 97,
```

Comment:
This is because we're now in a funny situation of having half dummy data, and half real data. the original dummy data was set between 0 and 10, now the data is between 97 and 99, so the `+97` is to centre the data at a reasonable range.